osu.Game/Graphics/Containers/Markdown/OsuMarkdownContainer.cs

```diff
@@ -21,6 +23,12 @@ public OsuMarkdownContainer()
             LineSpacing = 21;
         }
 
+        [BackgroundDependencyLoader]
+        private void load(IAPIProvider api)
+        {
+            DocumentUrl = api.WebsiteRootUrl;
+        }
+
```

Comment:
```suggestion
        protected override IReadOnlyDependencyContainer CreateChildDependencies(IReadOnlyDependencyContainer parent)
        {
            var api = parent.Get<IAPIProvider>();

            // needs to be set before the base BDL call executes to avoid invalidating any already populated markdown content.
            DocumentUrl = api.WebsiteRootUrl;

            return base.CreateChildDependencies(parent);
        }

```